packages/EasyBatch/src/Bridge/Symfony/Messenger/ProcessBatchItemMiddleware.php

```diff
@@ -45,6 +47,9 @@ public function handle(Envelope $envelope, StackInterface $stack): Envelope
             return $func();
         }
 
+        // Make sure to have a clean batch processor before processing item to prevent caching issues.
+        $this->batchProcessor->reset();
```

Comment:
This will not now work.
We need to clear the cache after the item is processing.

After `$this->processBatchItemWithLock` (line 57) we will have items in cache.